src/repl.jl

```diff
@@ -1,5 +1,14 @@
 
 promptname(level, name) = "$level|$name> "
+
+function write_prompt(terminal, mode)
+    @static if VERSION ≥ v"1.6.0-DEV.517"
```

Comment:
Which PR is this? Sorry if I broke Debugger...

Reply:
I don't think it was you. Some refactoring in how the REPL handled colors I think.

Reply:
Sounds good. Still might be good to link to the PR if the DEV number indicates you know it.

Reply:
No need to fix this PR, though, just for the future.